sdk/release/install.bat

```diff
@@ -1,5 +1,13 @@
 :: Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 :: SPDX-License-Identifier: Apache-2.0
 
-%~dp0daml\daml install %~dp0 --install-assistant=yes
+@SETLOCAL EnableDelayedExpansion
 
+@IF EXIST %~dp0daml_version.txt (
+    SET /P DAML_VERSION=<%~dp0daml_version.txt
+    SET CUSTOM_DAML_VERSION=--install-with-custom-version=!DAML_VERSION!
+)
```

Comment:
This was very tricky to get right. Originally I had `SET CUSTOM_DAML_VERSION=--install-with-custom-version=%DAML_VERSION%` as the second line, but that resulted in `-install-with-custom-version=` without the argument. The problem is that by default, `%SOME_VAR%` variables are inlined at _parse_ time (IF blocks are parsed all at once, not line by line) so `SET /P DAML_VERSION` above appeared to be ignored on the second line. The fix was to `@SETLOCAL EnableDelayedExpansion` _AND_ change `%DAML_VERSION%` to `!DAML_VERSION!`

https://ss64.com/nt/delayedexpansion.html